src/commands/runAIAgent.js

```diff
@@ -5,7 +5,8 @@ const { typeRealistically } = require('../util/realisticTyping');
 const { speak } = require('../util/speak');
 const { script } = require('../script');
 
-class SetupLayoutCommand extends Command {
+
+class runAgentCommand extends Command {
```

Comment:
By convention, class names start with a capital letter, to distinguish them from normal functions. So `RunAgentCommand` is more appropriate here.